.github/workflows/sphinx.yml

```diff
@@ -27,8 +27,6 @@ jobs:
     runs-on: ubuntu-24.04
     timeout-minutes: 10
     if: github.repository_owner == 'jupyter' || github.repository_owner == 'mathbunnyru' || github.event_name != 'schedule'
-    permissions:
-      contents: write
```

Comment:
This is wrong now - this was needed to support multi-lang docs and push strings to the main branch. We don't need that now.